### PR TITLE
dx(#914): use eslint-plugin-perfectionist to sort everything

### PR DESCRIPTION
**Pull request by Isokaeder**
Resolves one of the subtasks of #914

### CODE REVIEW

File: packages/eslint-config/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
 		"@typescript-eslint/utils": "^7.11.0",
 		"eslint-config-prettier": "^9.1.0",
 		"eslint-plugin-jsonc": "^2.16.0",
```
Comment (FlorianWendelborn) [inline packages/eslint-config/package.json:10]: needs version commit

File: packages/eslint-config/source/index.ts [hunk 2]
```diff
@@ -136,16 +137,15 @@ const baseConfig = tseslint.config({
 		'no-restricted-globals': ['error', ...noRestrictedGlobalsConfig],
 		'prefer-const': 'warn',
 		radix: 'error',
-		'valid-typeof': 'error',
-
 		// SonarJS
 		'sonarjs/no-collapsible-if': 'off', // replaced by unicorn/no-lonely-if
+
```
Comment (FlorianWendelborn) [inline packages/eslint-config/source/index.ts:142]: weird newline

File: packages/eslint-config/source/index.ts [hunk 3]
```diff
@@ -191,6 +191,7 @@ const baseConfig = tseslint.config({
 		'unicorn/prefer-type-error': 'warn',
 		'unicorn/require-array-join-separator': 'error',
 		'unicorn/require-number-to-fixed-digits-argument': 'warn',
+		'valid-typeof': 'error',
```
Comment (FlorianWendelborn) [inline packages/eslint-config/source/index.ts:194]: undo

File: packages/kotti-ui/source/kotti-card/KtCard.vue [hunk 4]
```diff
@@ -52,17 +52,17 @@ export default defineComponent({
 
 		return {
 			cardClass: computed(() => ({
+				[`kt-card--has-${props.imgPosition}-image`]: props.imgUrl,
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-card/KtCard.vue:55]: odd way to sort this

File: packages/kotti-ui/source/kotti-popover/KtPopover.vue [hunk 5]
```diff
@@ -193,15 +194,15 @@ export default defineComponent({
...
 						index,
 						option,
+						value,
 					})
 				},
 			KottiFieldToggle,
 			open,
-			showPopover,
 			triggerRef,
+			showPopover,
```
Comment (Isokaeder) [inline packages/kotti-ui/source/kotti-popover/KtPopover.vue:205]: ? s > t ?

File: packages/kotti-ui/source/kotti-field-select/components/OptionsItem.vue [hunk 6]
```diff
@@ -50,11 +50,11 @@ export default defineComponent({
 
 		return {
 			classes: computed(() => ({
+				[`kt-field-select-options-item--is-type-${props.type}`]: true,
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-field-select/components/OptionsItem.vue:53]: odd result too

File: packages/kotti-ui/source/kotti-navbar/KtNavbar.vue [hunk 7]
```diff
@@ -150,8 +151,8 @@ export default defineComponent({
 
 		return {
 			classes: computed(() => ({
-				'kt-navbar--narrow': props.isNarrow,
 				[`kt-navbar--theme-${props.theme}`]: true,
+				'kt-navbar--narrow': props.isNarrow,
```
Comment (Isokaeder) [inline packages/kotti-ui/source/kotti-navbar/KtNavbar.vue:155]: weird, are dynamic keys always sorted before static keys or something?

File: packages/kotti-ui/source/kotti-filters/KtFilters.vue [hunk 8]
```diff
@@ -111,9 +112,9 @@ export default defineComponent({
 
 		const searchColumn = computed(
 			() =>
-				(props.columns.find(
+				props.columns.find(
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-filters/KtFilters.vue:115]: what happened here?

File: packages/kotti-ui/source/kotti-filters/components/ButtonLink.vue [hunk 9]
```diff
@@ -51,9 +52,9 @@ export default defineComponent({
 	emits: ['click'],
 	setup(props, { emit }) {
 		const mainClasses = computed(() => ({
+			[`kt-button-link--type-${props.type}`]: !props.isDisabled,
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-filters/components/ButtonLink.vue:55]: odd result

File: packages/kotti-ui/source/kotti-navbar/KtNavbar.vue (hunk 7 above)
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-navbar/KtNavbar.vue:155]: odd result

File: packages/kotti-ui/source/kotti-popover/KtPopover.vue (hunk 5 above)
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-popover/KtPopover.vue:205]: uhm what

File: packages/kotti-ui/source/kotti-table-legacy/types.ts [hunk 10]
```diff
@@ -1,4 +1,8 @@
+/* eslint-disable perfectionist/sort-object-types */
+// TODO - separate groups by comments
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-table-legacy/types.ts:2]: image

File: packages/kotti-ui/source/types/kotti.ts [hunk 11]
```diff
@@ -7,31 +7,24 @@
  */
 import type { ZodSchema } from 'zod'
 
-export * from './decimal-separator'
-export * from './typed-emit'
-
 export { KottiAccordion as Accordion } from '../kotti-accordion/types'
 export { KottiActionbar as Actionbar } from '../kotti-actionbar/types'
-export { KottiAvatar as Avatar } from '../kotti-avatar/types'
+
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/types/kotti.ts:12]: suspicious newline

**Review by FlorianWendelborn — APPROVED** [review]
found a few minor oddities

**Pull request merged**